### DISCUSSION TITLE
这个X1,X2参数具体怎么写进去呀

### DISCUSSION THREAD

**Discussion by SoyKoa** [shuax/MouseInc.Settings#436] [Q&A]
["SendClick", "left"]
非常方便的发送鼠标点击，参数分别为left,right,middle,'x1','x2' 左键单击

比如我想发送左键双击, ["SendClick", "left",'x2' ]   是这样吗?

我现在是用的2次 ["SendClick", "left"] ,,有个问题是在看视频的时候,,,一般快速在视频窗口双击,,,会窗口最大化,,,如果是2次sendclick,,因为中间有时间间隔,,所以有时候可以最大化,,有时候又不行,,要看运气,,,,

所以老大,,想问下,,有没有什么代码 可以实现双击,,,而不是sendclick的方式连续点2次,,,,说明文档中的X1,X2,,,是代表点击的次数吗?

**Comment by Vermoudol** [comment]
试一下呗,咱也不知道
